ConfigurationValidation.hpp

```diff
@@ -6,6 +6,12 @@
 //                            ////////
 //////////////////////////////////////
 
+#define MIN_CONFIG_VERSION 1
+
+#if CONFIG_VERSION < MIN_CONFIG_VERSION
```

Comment:
If CONFIG_VERSION is not defined will this still be true?

Reply:
good catch, seems i forgot to commit this